packages/terra-framework-docs/src/terra-dev-site/doc/infinite-list/InfiniteList.1.doc.mdx

```diff
@@ -2,9 +2,9 @@ import { Badge } from 'terra-infinite-list/package.json?dev-site-package';
 import { Notice } from '@cerner/terra-docs';
 
 import InfiniteListSrc from 'terra-infinite-list/src/InfiniteList.jsx?dev-site-props-table';
-import InitialLoadingExample from '../example/InitialLoadingExample?dev-site-example';
-import ProgressiveLoadingExample from '../example/ProgressiveLoadingExample?dev-site-example';
-import VirtualizedExample from '../example/VirtualizedExample?dev-site-example';
+import InitialLoadingExample from './example/InitialLoadingExample?dev-site-example';
+import ProgressiveLoadingExample from './example/ProgressiveLoadingExample?dev-site-example';
+import VirtualizedExample from './example/VirtualizedExample?dev-site-example';
 
```

Comment:
I noticed many of the links on these docs are now broken because the link needs to be updated to point to "cerner-terra-core-docs". For example the terra-paginator link on line 26 below is now broken. It should reference `components/cerner-terra-core-docs/paginator/paginator`. Can you please update it and any others that you see?